add parameter for ordered script execution

aFarkas opened the pull request:
This should replace PR https://github.com/filamentgroup/loadJS/pull/14. It is an implementation of the ordered execution feature without hurting the old API. As discussed there. You might want to critically review my readme changes.

zachleat commented:
Hey @scottjehl after reading the discussion on #14 I think this might be worth a merge. If you’re onboard I’ll merge it.

@aFarkas you still ok with this method?

aFarkas commented:
@zachleat 
Yes I'm still fine with this method. I made some changes to the readme. (I removed the `requestAnimationFrame` advice (You shouldn't use that.) And also removed the advice to not use ordered execution (simply because of missing IE9- support).

scottjehl commented:
Quite late on this, but it should be in the code base. I'll merge and get a new release out.
